oci/private/pull.bzl

```
@@ -312,7 +312,7 @@ def _get_auth_config_path(rctx):
         path = rctx.read(rctx.path(rctx.attr.config_path))
 
     if path:
-        return json.decode(rctx.read(path))
+        return json.decode(rctx.read(path).strip())
```

```suggestion
        return json.decode(rctx.read(path.strip()))
```
Do you mean to strip newlines on the path itself instead?
json decode should already be able to handle trailing newlines